### PR TITLE
Added multiple tabs on the settings page

### LINKED ISSUE #235: To organize the settings page with multiple tabs rather than a continuous page
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
The settings page is a continuous page

**Expected behavior:**
To organize the settings page with multiple tabs rather than a continuous feature .This will make the settings page more appealing and more organized.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #235 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
organizes the settings page with multiple tabs rather than a continuous feature

#### Changes proposed in this pull request:
Added multiple tabs

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@ms10398 @zamhaq @gabru-md

**Comment by stealthanthrax** [comment]
screen shot 2017-12-25 at 8 03 51 pm

**Comment by stealthanthrax** [comment]
@ms10398 , have a look now

**Comment by stealthanthrax** [comment]
@0x48piraj what conflicts?

**Comment by stealthanthrax** [comment]
done @ms10398  @0x48piraj

**Comment by stealthanthrax** [comment]
@zamhaq ,those commits were lost in a force push after squashing

**Comment by ms10398** [comment]
Then check it again and please add it back @stealthanthrax

**Comment by stealthanthrax** [comment]
@zamhaq @ms10398 , its working now

**Comment by stealthanthrax** [comment]
screen shot 2017-12-30 at 1 49 48 am

**Comment by stealthanthrax** [comment]
@ms10398 ,have a look now

File: src/options.html [hunk 1]
```diff
@@ -67,7 +78,11 @@ <h3>Settings Page</h3>
                 </div>
             </div>
             <div class="row">
+<<<<<<< HEAD
```
Comment (akshatnitd) [inline src/options.html]: @stealthanthrax Check some extra lines have to removed that may have come due to the conflict. Please check them :)

**Comment by stealthanthrax** [comment]
@ms10398 @zamhaq @gabru-md , updated have a look

**Comment by ms10398** [comment]
Whats the status @stealthanthrax

**Comment by stealthanthrax** [comment]
have a look now @ms10398

**Comment by ms10398** [comment]
@gabru-md Have a look at it

**Comment by stealthanthrax** [comment]
@ms10398 ,  I am not having any issues , please elaborate

**Comment by ms10398** [comment]
Check your console of the extension when you open the extension.
You have tried to access `nav` element which is present in `options` page and not present when the extension is opened which creates console errors.

**Comment by stealthanthrax** [comment]
screen shot 2018-01-05 at 3 46 01 am

i am facing only this in the console,please could you add a screenshot?

**Comment by ms10398** [comment]
@gabru-md Can you have a look of the PR in your console :+1:

**Comment by gabru-md** [comment]
@stealthanthrax can you update here?
or can this be closed?

**Comment by prateekiiest** [comment]
@stealthanthrax could you resolve those console errors in your local machine

**Comment by ms10398** [comment]
Closing it due to inactivity

**Pull request closed without merging**